test/extensions/filters/http/grpc_http1_reverse_bridge/reverse_bridge_test.cc

```diff
@@ -801,6 +802,198 @@ TEST_F(ReverseBridgeTest, RouteWithTrailers) {
   }
 }
 
+// Verifies that the filter streams responses when it's configured to set the content length based
+// on a header returned from the upstream.
+TEST_F(ReverseBridgeTest, WithholdGrpcStreamResponse) {
```

Comment:
can this be an integration test too?  for HTTP/2 I hope saying "content-length:20" sending 20 bytes and a reset would result in reset rather than "stream complete" but I'd like to have that tested :-)